client/src/Setup/ProgramDirector.js

```diff
@@ -1,10 +1,213 @@
 import { useAuthContext } from "../AuthContext";
+import './ProgramDirector.scss';
+
+import { useState, useEffect } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import Api from "../Api";
 
 function ProgramDirector() {
   const { user } = useAuthContext();
+  const history = useHistory();
+  const [organization, setOrganization] = useState({
+    name: '',
+    type: '',
+    phoneNumber: '',
+    email: '',
+    address: '',
+    country: '',
+    state: '',
+    city: '',
+    description: '',
+    partnershipDescription: ''
+  });
+
+  function onChange(event) {
+    const newOrganization = { ...organization };
+    newOrganization[event.target.name] = event.target.value;
+    setOrganization(newOrganization);
+  }
+
+  async function onSubmit(event) {
+    event.preventDefault();
+    try {
+      const response = await Api.organizations.create(organization);
```

Comment:
When we're in this setup flow, we're always going to be creating an organization, so I removed the code related to looking for an id and loading/updating an existing organization. There's no :id path parameter in the route for this page...

Notice how in the next line, I use the data coming back from the organizations.create call to create the programDirector association between the logged in user and the newly created organization.

Calling this API doesn't actually require an admin user right now... The reason why you need to be an Admin in the  CRUD form I wrote is because that form wants to load all users and organizations into the dropdowns of the form- and being able to fetch/see all users/orgs requires Admin privileges. Here, however, we know in code what UserId/OrganizationId we are going to use so we can go ahead do so...

Note that, in a real production app, we would never actually do two API calls like this one right after each other- this introduces the possibility for something to break between the two calls, and it really should not be possible for anyone to arbitrarily create new program directors.

Rather, we'd probably create a new, dedicated setup route in the Express API backend that would do both actions in one step- create a new Organization, and add the creating User as the first ProgramDirector. Note, however, that the resulting records in the database would be the same...

So I think this is fine for now for prototyping purposes...